### PR TITLE
About Us Section

### PR DESCRIPTION
**Pull request by LadaM**
layout and styling for about section

Please note: 
* added Typography region to the base.css with .section-heading and .bold styles that can be used everywhere
* changed the basic font style in the body, updated the line-height, color (it is lighter at 50% saturation) and letter-spacing

### CODE REVIEW

**Comment by LadaM** [comment]
Mobile view:
image
Tablet view:
Screenshot 2024-03-27 at 09 23 35
Desktop view:
Screenshot 2024-03-27 at 09 26 04

File: src/css/base.css [hunk 1]
```diff
@@ -136,3 +140,20 @@ button {
...
+/* #region Typography */
+.section-heading {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 36px;
+  text-align: center;
+  color: var(--black);
+}
+
+.bold {
```
Comment (kazamov) [inline src/css/base.css:153]: The meaning of `bold` is to provide just `font-weight`. Maybe it should have a different name e.g. `heading-small` or `small-heading`. Also I checked the styles, you missed the `line-height: 1.2; letter-spacing: -0.4px;` for that heading. Please update the class name and add missed properties.

File: src/css/about.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+@media screen and (min-width: 1280px) {
+    .about-section {
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
+
+    /* the width of the list item container should be 2/3 */
+    .about-section .item-container {
+        flex-basis: calc(((100% - 64px) / 3) * 2);
```
Comment (kazamov) [inline src/css/about.css:85]: Just a recommendation, the Grid CSS solution is simpler. But up to you.

File: src/css/base.css [hunk 3]
```diff
@@ -24,8 +26,10 @@ body {
   display: flex;
   flex-direction: column;
   font-size: 14px;
+  line-height: 24px;
```
Comment (kazamov) [inline src/css/base.css:29]: Line height should be set as a float number

**Review by kazamov — CHANGES_REQUESTED** [review]

File: src/css/base.css (hunk 1 above)
Comment (LadaM) [reply src/css/base.css:153]: The line height and letter spacing is the same as for the normal text that is why I changed the body styling.

File: src/css/base.css (hunk 3 above)
Comment (LadaM) [reply src/css/base.css:29]: Here I had a discussion even when we were doing homework. I don’t like the values like 171.43% or 1.72. They end up giving imprecise sizes. Relative Line Height = (24px / 14px) * 100%
                      ≈ 171.43%

Comment (kazamov) [reply src/css/base.css:29]: Better to be consistent as there is already relative line height used in other places.

**Review by kazamov — APPROVED** [review]

**Pull request merged**